packages/headless/src/features/configuration/configuration-slice.ts

```diff
@@ -29,7 +33,9 @@ export const configurationReducer = createReducer(
         if (action.payload.platformUrl) {
           state.platformUrl = action.payload.platformUrl;
           state.search.apiBaseUrl = `${action.payload.platformUrl}${searchAPIEndpoint}`;
-          state.analytics.apiBaseUrl = `${action.payload.platformUrl}${analyticsAPIEndpoint}`;
+          state.analytics.apiBaseUrl = `${analyticsUrlFromPlatformUrl(
```

Comment:
Could one have a proxy, which on their end is the same for the platform & analytics, simply with different routes, would we handle those? 
Maybe `analyticsUrlFromPlatformUrl` should be more specific to a Coveo Url (not just starting by "https//platform") and if it doesn't match, we default to the past behaviour of only changing the route